lib/parseMessageDefinition.js

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2018-present, GM Cruise LLC
+
+// This source code is licensed under the Apache License, Version 2.0,
+// found in the LICENSE file in the root directory of this source tree.
+// You may not use this file except in compliance with the License.
+
+const builtins = new Set([
+  "string",
+  "bool",
+  "int8",
+  "uint8",
+  "int16",
+  "uint16",
+  "int32",
+  "uint32",
+  "float32",
+  "float64",
+  "int64",
+  "uint64",
+  "time",
+  "duration",
+]);
```

Comment:
This might be nice to export, as it's a useful list that I've seen duplicated in our internal projects.

Reply:
exported as rosPrimitiveTypes in index.js